File: scholar-web/src/main/webapp/app/model/transport/vehicle/VehicleSearch.js
Ext.define('scholar.model.transport.vehicle.VehicleSearch', {
	extend: 'Ext.data.Model',
	idProperty:'id',
	fields : [ {
		name: 'id',
		type: 'int'
	},{
		name : 'vehicleNumber',
		type : 'string'
	}, {
		name : 'vehicleDetails',
		type : 'string'
	}, {
		name : 'lastUpdatedDate',
		type : 'date',
		dateFormat : 'F D Y'
	} ]
});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/VehicleSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class VehicleSearch extends AbstractEntity {
	int vehicleId;
	String vehicleNumber;
	String vehicleDetails;
	@Temporal(TemporalType.DATE)
	Date lastUpdatedDate;
}
